website_upload_video/README.rst

```diff
@@ -0,0 +1,66 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=======================
+Upload video on website
+=======================
+
+This module adds a "Upload video" option to the "Select a Media / Video" dialog.
```

Comment:
Please explain when is useful this functionality as Odoo has a standard for showing videos. Also IMHO could be very interesting to notice that is you use this functionality instead of Odoo's one, the server where is running Odoo must have enough storage for all videos (of course I suppose that videos are not stored in database). Not everyone realizes this. Thanks!